src/prompts/actions.ts

```diff
@@ -0,0 +1,81 @@
+import { Octokit } from "@octokit/rest";
+import inquirer from "inquirer";
+
+import actionBuilders, { Action, ActionName, Actions } from "../actions";
+
+type Result = {
+  targetOwner: string;
+  actions: ActionName[];
+};
+
+const choices = [
+  {
+    name: "Close all issues",
+    value: "closeIssues",
+    short: "Close issues",
+  },
+  {
+    name: "Close all pull requests",
+    value: "closePullRequests",
+    short: "Close pull requests",
+  },
+  {
+    name: "Update repository (visibility, archive, etc.)",
+    value: "update",
+    short: "Update",
+  },
+  {
+    name: "Transfer repository",
+    value: "transfer",
+    short: "Transfer",
+  },
+];
+
+if (process.env.ENABLE_DELETE === "true") {
+  choices.unshift({
+    name: "Delete repository",
+    value: "delete",
+    short: "Delete",
+  });
+}
+
+const buildQuestions = (user: string): inquirer.QuestionCollection<Result> => [
+  {
+    type: "input",
+    name: "targetOwner",
+    message: "Which organization do you want to organize?",
+    default: user,
+  },
+  {
+    type: "checkbox",
+    name: "actions",
+    message: "Choose actions.",
+    choices: choices,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/nownabe/ghrepos-organizer/pullRequest?prid=9051926)
```suggestion
    choices,
```